src/Symfony/Component/Intl/Resources/data/timezones/rw.json

```diff
@@ -3,20 +3,8 @@
     "Names": {
         "Africa\/Kigali": "U Rwanda (Kigali)",
         "Antarctica\/Troll": "Troll",
-        "CST6CDT": "CST6CDT",
```

Comment:
we should never provide zone IDs as translations, in this case the runtime fallback should apply instead.